integration_tests/features/support/steps.js

```diff
@@ -766,4 +792,51 @@ Then('Difficulties are available', function () {
 
 });
 
+When(/I coin split tari in wallet (.*) to produce (.*) UTXOs of (.*) uT each with fee_per_gram (.*) uT/, {timeout: 25*5*1000}, async function (walletName, splitNum, splitValue, feePerGram) {
+    console.log("\n");
+    let wallet = this.getWallet(walletName)
+    let walletClient = wallet.getClient();
+    let walletInfo = await walletClient.identify();
+    let result = await walletClient.coin_split(
+        {
+            "amount_per_split": splitValue,
+            "split_count": splitNum,
+            "fee_per_gram": feePerGram,
+            "message": "Cucumber coinsplit",
+            "lockheight": 0,
+        }
+    );
+    console.log("Coin split to produce", splitNum, "values of", splitValue, "uT completed with TxId: ", result);
+    this.addTransaction(walletInfo["public_key"], result["tx_id"]);
+    this.lastResult = result;
+});
+
+When(/I send (.*) transactions of (.*) uT each from wallet (.*) to wallet (.*) at fee_per_gram (.*)/, {timeout: 25*5*1000}, async function (numTransactions, amount, sourceWallet, dest, fee) {
```

Comment:
```suggestion
When(/I send (.*) transactions of (.*) uT each from wallet (.*) to wallet (.*) at fee_per_gram (.*)/, {timeout: 1200*1000}, async function (numTransactions, amount, sourceWallet, dest, fee) {
```

Reply:
Honestly I feel that these timeouts should actually be short because these components will be used by other tests where we want to enforce a short timeout. So lengthening them for the stress test I don't think is correct. If you need them long which using the stress test to find an issue then you can update them.